challenges/5-arraycardio/arraycardio.js

```diff
@@ -72,7 +100,23 @@ function findIndex(arr, num) {
 
 */
 function getLetterGrades(arr) {
-    // implementation
+    letterGrade();
+    let newArr = [];
+    for (let i = 0; i < arr.length; i++) {
+        if (arr[i] == "A") {
+            newArr.push(arr[i]);
+        } else if (arr[i] == "B") {
+            newArr.push(arr[i]);
+        } else if (arr[i] == "C") {
+            newArr.push(arr[i]);
+        } else if (arr[i] == "D") {
+            newArr.push(arr[i]);
+        } else if (arr[i] == "F") {
+            arr[i] = "F";
+            newArr.push(arr[i]);
+        }
+    }
+    return newArr;
```

Comment:
Did this pass the tests? If so I wrote them wrong. Go through this line by line and see what would happen for `getLetterGrades([95,85])`

Reply:
Ah, I didn't write tests for this lmao